pkg/console/subresource/deployment/deployment_test.go

```diff
@@ -9,20 +9,101 @@ import (
 	corev1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/resource"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
 
 	configv1 "github.com/openshift/api/config/v1"
 	operatorsv1 "github.com/openshift/api/operator/v1"
 	"github.com/openshift/console-operator/pkg/api"
+	"github.com/openshift/console-operator/pkg/console/assets"
 	"github.com/openshift/console-operator/pkg/console/subresource/configmap"
 	"github.com/openshift/console-operator/pkg/console/subresource/util"
+	"github.com/openshift/library-go/pkg/operator/resource/resourceread"
 )
 
+const (
+	workloadManagementAnnotation      = "target.workload.openshift.io/management"
+	workloadManagementAnnotationValue = `{"effect": "PreferredDuringScheduling"}`
+)
+
+func consoleContainer(cr *operatorsv1.Console, volConfigList []volumeConfig, proxyConfig *configv1.Proxy) corev1.Container {
```

Comment:
I dont think we should need this function anymore. Its kinda redundant to the deployment's bindata

Reply:
@jhadvig agreed; removed!